src/actions/bundle.ts

```
@@ -0,0 +1,53 @@
+import path from "path";
+import fs from "fs";
+import { GraphAILogger } from "graphai";
+import { MulmoPresentationStyleMethods } from "../methods/index.js";
```

ESLint

[error] Remove this unused import of 'MulmoPresentationStyleMethods'.

(sonarjs/unused-import)